Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,46 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+    
+# Do you care about the environment?
+default persistent._mas_pm_cares_environment = None
+
+init 5 python:
+    addEvent(
+        Event(persistent.event_database,eventlabel="monika_environment",category=['nature', 'life'],prompt="The environment",random=True))
+ 
+label monika_environment:
+    m 2eud "Hey [player], I've been wondering..."
+    $ _history_list.pop()
+    menu:
+        m "Do you care about the environment{fast}?"
+        "Yes.":
+            $ persistent._mas_pm_cares_environment = True
+            m 3sub "That's wonderful!"
+            m 3hsb "I'm so glad to hear that, player."
+        
+        "No.":
+            $ persistent._mas_pm_cares_environment = False
+            m 1rkb "That's alright, you're entitled to your opinion."
```

Comment:
When you say this, it makes it seem like Monika disagrees, and just doesn't want to argue with you about it. Kind of like "I'll just go my way and you go yours, and we never have to discuss again". 

I think she should try to be a bit more understanding, and maybe try and justify why you might think that way, instead of just dismissing your opinion and going on about her own thoughts.